### ISSUE TITLE
Not clear why build is breaking

### ISSUE THREAD

**Issue by cowwoc** [siom79/japicmp#151]
If you look at the attached report, it's not clear why japicmp is failing the build. As far as I can tell, the report contains the following changes:

* New interfaces added.
* New methods added to concrete classes.

Neither of these should break source or binary compatibility.

[japicmp.pdf](https://github.com/siom79/japicmp/files/518125/japicmp.pdf)

**Comment by siom79** [comment]
Are you using the latest [version](http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22japicmp%22%20AND%20v%3A%220.9.1%22)?

Newer versions of japicmp print out the exact reason for a source or binary incompatible change in the XML (for example):
```
<compatibilityChanges>
    <compatibilityChange>SUPERCLASS_CHANGED</compatibilityChange>
</compatibilityChanges>
```
Can you provide the XML file for the latest version? Thanks in advance.

**Comment by cowwoc** [comment]
@siom79 Yes, please reopen this issue. I am using version 0.9.1. Unfortunately Github won't allow me to attach XML files (even ZIP doesn't work for some reason). Please download the file from https://bitbucket.org/cowwoc/requirements/downloads/japicmp.xml and attach it to this issue on my behalf.

The "cause" message I get starts with this text `Breaking the build because there is at least one incompatibility: org.bitbucket.cowwoc.requirements.AssertionVerifier.addContext(java.lang.String,java.lang.Object):METHOD_NOW_THROWS_CHECKED_EXCEPTION`

It goes on to list many more entries, but as I mentioned above all of them seem to be related to the addition of new interfaces that didn't exist before. https://github.com/siom79/japicmp/issues/112 was supposed to fix this problem. Perhaps we are looking at a regression?

**Comment by siom79** [comment]
Thanks for providing the details. It helped me to reconstruct your case. I could fix it on the `develop` branch and it will be released with 0.9.2.

**Comment by cowwoc** [comment]
Sounds good. I'll keep my eyes open for that release. Thanks!

**Comment by cowwoc** [comment]
@siom79 I tested the `develop` branch and the fix worked for me. Thanks!

**Comment by siom79** [comment]
Released with 0.9.2.
